src/lib.rs

```diff
@@ -35,12 +35,14 @@ pub struct AccountingAlloc<A = System> {
 }
 
 #[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
-/// A count of allocated and deallocated bytes returned by [`AccountingAlloc::count`].
+/// Statistics for allocations and deallocations made with an [`AccountingAlloc`].
 pub struct AllocCounts {
-    /// Count of allocated bytes.
+    /// Count of all bytes ever allocated with the allocator, globally.
     pub alloc: usize,
-    /// Count of deallocated bytes.
+    /// Count of all bytes ever deallocated with the allocator, globally.
     pub dealloc: usize,
+    /// Largest allocation size in bytes since the last call to [`AccountingAlloc::count`].
+    pub largest_alloc: usize,
```

Comment:
It now seems odd to me that `alloc` and `dealloc` are running totals but `largest_alloc` is incremental. Maybe they should all be incremental?

Reply:
True, will change! I was going to say it would double contention on each alloc/dealloc counter, but if we keep track of the last total in `ThreadCountersShared`, we can do it without additional overhead.

Reply:
fixed in 3e21a91